grpc/interceptors_test.go

```diff
@@ -162,8 +162,8 @@ func TestTimeouts(t *testing.T) {
 		expectedErrorPrefix string
 	}{
 		{250 * time.Millisecond, "rpc error: code = Unknown desc = rpc error: code = DeadlineExceeded desc = the chiller overslept"},
-		{100 * time.Millisecond, "rpc error: code = DeadlineExceeded desc = not enough time left on clock: "},
-		{10 * time.Millisecond, "rpc error: code = DeadlineExceeded desc = not enough time left on clock: "},
+		{100 * time.Millisecond, "Chiller.Chill timed out after 0 ms"},
```

Comment:
I think this might be flaky or the wrong expected output. In CI the `TestTimeouts` unit test failed in both `config` and `config-next` with output like:
```

--- FAIL: TestTimeouts (0.24s)
    --- FAIL: TestTimeouts/100ms (0.00s)
        interceptors_test.go:178: Wrong error. Got Chiller.Chill timed out after 1 ms, expected Chiller.Chill timed out after 0 ms
```